### PR TITLE
add compliance memo link and edit some language

### PR DESCRIPTION
**Pull request by wslack**
Merge with https://github.com/18F/federalist-docs/pull/224

### CODE REVIEW

**Comment by wslack** [comment]
I don't think I need to edit https://github.com/18F/federalist/blob/wslack-patch-1/test/api/requests/main.test.js , but not 100% sure

**Comment by wslack** [comment]
Removing 
`          <p>Federalist takes care of the heavy lifting for federal web hosting and compliance, allowing you to focus your expertise on your content.</p>
`
Added `<h2><a id="page-body"></a>Federalist takes care of the heavy lifting</h2>` back in

File: views/home.njk [hunk 1]
```diff
@@ -7,24 +7,25 @@
...
-            Rapidly launch a compliant website for your federal agency, program or office
+            Rapidly launch a compliant website for your federal agency, program, or office
           </h1>
         </div>
         <div class="usa-width-one-third usa-hero-callout">
           <hr class="hr-dark"></hr>
           <h5 class="contrast-heading">Federalist Stats:</h5>
           <div class="usa-hero-callout-stats">
+            <p>119 live .gov sites supported</p>
+            <p>8 federal agency customers</p>
```
Comment (annahsebok) [inline views/home.njk]: I thought we were going with `partners`

**Comment by annahsebok** [comment]
Made one small note (partners vs. customers) but other than that, LGTM!

**Comment by wslack** [comment]
ah - I chose "customers" here because the term applied to the entire agency. "Partners" are what I would call people that log into Federalist to manage sites.

"Agency Partners" seems unclear - as if we were working on Federalist with those agencies.

**Comment by annahsebok** [comment]
@wslack Yes, that makes sense. I'd suggest using "partners" to retain consistency, but given the fact that we're not yet working with the folks mentioned here, I'd suggest "potential agency partners" over "federal agency customers." Varying nouns can be disorienting and makes the way we talk about the people we work with or have the potential to work with inconsistent.

**Comment by wslack** [comment]
Now I'm more confused. We work with 8 agencies - and are working actively with all of those agencies. It's just that the "partners" in those agencies - the civil servants - are only sometimes in an office or two within the whole agency.

**Comment by annahsebok** [comment]
@wslack and I synced up on this — we're going to leave this as `customers` — `partners` is a term that we have more context around internally and could confuse folks about the relationships we have with agencies. I'll be talking a broader look at this when I'm working on #1685.

**Comment by annahsebok** [comment]
In the meantime — @wslack this is 👍 from me!

**Pull request merged**